### PR TITLE
SIMD i8x16.shl implementation.

### PR DESCRIPTION
**Pull request by lizhengxing**

### CODE REVIEW

**Comment by lizhengxing** [comment]
@binji 
i8x16.shl implementation, PTAL!
Thanks!

@arunetm FYI!

File: src/wast-lexer.cc [hunk 1]
```diff
@@ -543,6 +543,7 @@ Token WastLexer::GetToken(WastParser* parser) {
       <i> "i8x16.sub_saturate_u" { RETURN_OPCODE(Binary, I8X16SubSaturateU); }
       <i> "i16x8.sub_saturate_s" { RETURN_OPCODE(Binary, I16X8SubSaturateS); }
       <i> "i16x8.sub_saturate_u" { RETURN_OPCODE(Binary, I16X8SubSaturateU); }
+      <i> "i8x16.shl" 		 { RETURN_OPCODE(Binary, I8X16Shl); }
```
Comment (binji) [inline src/wast-lexer.cc]: nit: align { please!

File: src/interp.cc [hunk 2]
```diff
@@ -2455,6 +2455,14 @@ Result Thread::Run(int num_instructions) {
       case Opcode::I16X8SubSaturateU:
         CHECK_TRAP(SimdBinop<v128, uint16_t>(SubSaturate<int32_t, uint16_t>));
         break;
+
+      case Opcode::I8X16Shl: {
+        uint32_t shift_data = Pop<uint32_t>();
```
Comment (binji) [inline src/interp.cc]: nit: `shift_count` instead, perhaps?

File: src/interp.cc [hunk 3]
```diff
@@ -2956,6 +2964,12 @@ void Thread::Trace(Stream* stream) {
       break;
     }
 
+    case Opcode::I8X16Shl: { 
+      stream->Writef("%s $0x%08x %08x %08x %08x  $0x%08x\n", opcode.GetName(), Pick(2).v128_bits.v[0],
+                       Pick(2).v128_bits.v[1], Pick(2).v128_bits.v[2], Pick(2).v128_bits.v[3],Pick(1).i32);
```
Comment (binji) [inline src/interp.cc]: nit: wrap line at 80 characters

File: src/interp.cc (hunk 2 above)
Comment (lizhengxing) [reply src/interp.cc]: Done.

File: src/interp.cc (hunk 3 above)
Comment (lizhengxing) [reply src/interp.cc]: Done

File: src/wast-lexer.cc (hunk 1 above)
Comment (lizhengxing) [reply src/wast-lexer.cc]: Done.

**Pull request merged**